--- seamonkey-2.5/mail/app/profile/all-thunderbird.js.old	2011-11-21 11:20:08.000000000 +0100
+++ seamonkey-2.5/mail/app/profile/all-thunderbird.js	2011-12-14 12:40:24.074179826 +0100
@@ -138,7 +138,7 @@ pref("extensions.update.autoUpdateDefaul
 // Disable add-ons installed into the shared user and shared system areas by
 // default. This does not include the application directory. See the SCOPE
 // constants in AddonManager.jsm for values to use here
-pref("extensions.autoDisableScopes", 15);
+pref("extensions.autoDisableScopes", 0);
 
 // Preferences for AMO integration
 pref("extensions.getAddons.cache.enabled", true);
--- iceape-2.22/suite/browser/browser-prefs.js~	2013-11-06 19:54:38.789734061 +0100
+++ iceape-2.22/suite/browser/browser-prefs.js	2013-11-06 20:55:33.765611935 +0100
@@ -616,7 +616,7 @@
 // Disable add-ons installed into the shared user and shared system areas by
 // default. This does not include the application directory. See the SCOPE
 // constants in AddonManager.jsm for values to use here.
-pref("extensions.autoDisableScopes", 15);
+pref("extensions.autoDisableScopes", 3);
 
 // Preferences for AMO integration
 pref("extensions.getAddons.cache.enabled", true);  // also toggles personalized recommendations
--- iceape-2.22/mozilla/browser/app/profile/firefox.js~	2013-10-29 02:20:40.000000000 +0100
+++ iceape-2.22/mozilla/browser/app/profile/firefox.js	2013-11-06 20:56:58.633092070 +0100
@@ -62,7 +62,7 @@
 
 // Disable add-ons that are not installed by the user in all scopes by default.
 // See the SCOPE constants in AddonManager.jsm for values to use here.
-pref("extensions.autoDisableScopes", 15);
+pref("extensions.autoDisableScopes", 3);
 
 // Dictionary download preference
 pref("browser.dictionaries.download.url", "https://addons.mozilla.org/%LOCALE%/firefox/dictionaries/");
