--- mozilla/suite/browser/browser-prefs.js	2006-12-22 14:51:33.000000000 +0100
+++ mozilla/suite/browser/browser-prefs.js	2008-06-24 21:50:31.000000000 +0200
@@ -378,12 +378,12 @@
 // App-specific update preferences
 
 // Whether or not app updates are enabled
-pref("app.update.enabled", true);
+pref("app.update.enabled", false);
  
 // This preference turns on app.update.mode and allows automatic download and
 // install to take place. We use a separate boolean toggle for this to make
 // the UI easier to construct.
-pref("app.update.auto", true);
+pref("app.update.auto", false);
 
 // Defines how the Application Update Service notifies the user about updates:
 //
