--- seamonkey-1.1.10/configure.in.orig	2008-06-16 00:25:32.000000000 +0200
+++ seamonkey-1.1.10/configure.in	2008-07-10 18:30:45.487028766 +0200
@@ -3746,7 +3746,7 @@
 fi
 
 if test -n "$MOZ_NATIVE_NSS"; then
-   NSS_LIBS="$NSS_LIBS -lcrmf"
+   NSS_LIBS="-lcrmf $NSS_LIBS"
 else
    NSS_CFLAGS='-I$(LIBXUL_DIST)/include/nss'
    NSS_DEP_LIBS="\
@@ -7629,7 +7629,7 @@
         mkdir "directory/c-sdk/ldap"    
     fi
 
-    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --without-nss --with-mozilla"
+    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-nss --with-mozilla"
     if test -z "$MOZ_DEBUG"; then
         ac_configure_args="$ac_configure_args --disable-debug"
     fi
--- seamonkey-1.1.10/configure.orig	2008-06-16 00:25:31.000000000 +0200
+++ seamonkey-1.1.10/configure	2008-07-10 18:31:04.379027928 +0200
@@ -11842,7 +11842,7 @@
 fi
 
 if test -n "$MOZ_NATIVE_NSS"; then
-   NSS_LIBS="$NSS_LIBS -lcrmf"
+   NSS_LIBS="-lcrmf $NSS_LIBS"
 else
    NSS_CFLAGS='-I$(LIBXUL_DIST)/include/nss'
    NSS_DEP_LIBS="\
@@ -21195,7 +21195,7 @@
         mkdir "directory/c-sdk/ldap"    
     fi
 
-    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --without-nss --with-mozilla"
+    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-nss --with-mozilla"
     if test -z "$MOZ_DEBUG"; then
         ac_configure_args="$ac_configure_args --disable-debug"
     fi
