diff -urN mozilla.orig/editor/ui/locales/en-US/chrome/region/region.properties mozilla/editor/ui/locales/en-US/chrome/region/region.properties
--- mozilla.orig/editor/ui/locales/en-US/chrome/region/region.properties	2009-10-17 15:48:19.000000000 +0200
+++ mozilla/editor/ui/locales/en-US/chrome/region/region.properties	2009-10-24 13:36:13.000000000 +0200
@@ -1,4 +1,4 @@
 #
 # editor.js
 # editor.properties
-editor.throbber.url=http://www.seamonkey-project.org/
+editor.throbber.url=http://www.pld-linux.org/Packages/Iceape
diff -urN mozilla.orig/mozilla/extensions/reporter/resources/content/reporter/reportWizard.js mozilla/mozilla/extensions/reporter/resources/content/reporter/reportWizard.js
--- mozilla.orig/mozilla/extensions/reporter/resources/content/reporter/reportWizard.js	2009-10-17 15:53:53.000000000 +0200
+++ mozilla/mozilla/extensions/reporter/resources/content/reporter/reportWizard.js	2009-10-24 13:36:13.000000000 +0200
@@ -413,7 +413,7 @@
   catch(ex) {}
   // only works on Gecko 1.8 and higher (if above doesn't return)
   if ('nsIChromeRegistrySea' in Components.interfaces) {
-    return 'SeaMonkey/'+
+    return 'Iceape/'+
     Components.classes['@mozilla.org/network/io-service;1']
               .getService(Components.interfaces.nsIIOService)
               .getProtocolHandler('http')
diff -urN mozilla.orig/mozilla/toolkit/content/about.xhtml mozilla/mozilla/toolkit/content/about.xhtml
--- mozilla.orig/mozilla/toolkit/content/about.xhtml	2009-10-17 15:54:53.000000000 +0200
+++ mozilla/mozilla/toolkit/content/about.xhtml	2009-10-24 13:36:13.000000000 +0200
@@ -68,9 +68,6 @@
   <ul id="aboutPageList">
     <li>&about.copy.beforeLink; <a href="about:credits">&about.copy.linkTitle;</a> &about.copy.afterLink;</li>
     <li>&about.license.beforeLink; <a href="about:license">&about.license.linkTitle;</a> &about.license.afterLink;</li>
-    <!-- Release notes link removed pending fix for bug 349985
-    <li>&about.relnotes.beforeLink; <a id="releaseNotesURL" href="">&about.relnotes.linkTitle;</a> &about.relnotes.afterLink;</li>
-    -->
     <li>&about.buildconfig.beforeLink; <a href="about:buildconfig">&about.buildconfig.linkTitle;</a> &about.buildconfig.afterLink;</li>
     <script type="application/javascript">
       // get release notes URL from prefs
diff -urN mozilla.orig/mozilla/toolkit/xre/nsAppRunner.cpp mozilla/mozilla/toolkit/xre/nsAppRunner.cpp
--- mozilla.orig/mozilla/toolkit/xre/nsAppRunner.cpp	2009-10-17 15:55:06.000000000 +0200
+++ mozilla/mozilla/toolkit/xre/nsAppRunner.cpp	2009-10-24 13:36:13.000000000 +0200
@@ -1236,8 +1236,7 @@
 static inline void
 DumpVersion()
 {
-  printf("%s %s %s", 
-         gAppData->vendor ? gAppData->vendor : "", gAppData->name, gAppData->version);
+  printf("%s %s", gAppData->name, gAppData->version);
   if (gAppData->copyright)
       printf(", %s", gAppData->copyright);
   printf("\n");
diff -urN mozilla.orig/mozilla/xpfe/bootstrap/appleevents/mozilla.sdef mozilla/mozilla/xpfe/bootstrap/appleevents/mozilla.sdef
--- mozilla.orig/mozilla/xpfe/bootstrap/appleevents/mozilla.sdef	2009-10-17 15:55:12.000000000 +0200
+++ mozilla/mozilla/xpfe/bootstrap/appleevents/mozilla.sdef	2009-10-24 13:36:13.000000000 +0200
@@ -8,7 +8,7 @@
 -->
 
 <dictionary title="">
-	<suite name="SeaMonkey" code="MOZZ">
+	<suite name="Iceape" code="MOZZ">
 
 	</suite>
 
diff -urN mozilla.orig/suite/app/application.ini mozilla/suite/app/application.ini
--- mozilla.orig/suite/app/application.ini	2009-10-17 15:48:25.000000000 +0200
+++ mozilla/suite/app/application.ini	2009-10-24 13:36:13.000000000 +0200
@@ -37,8 +37,8 @@
 
 #filter substitution
 [App]
-Vendor=Mozilla
-Name=SeaMonkey
+Vendor=
+Name=Iceape
 Version=@APP_VERSION@
 BuildID=@GRE_BUILDID@
 #ifdef MOZ_SOURCE_REPO
diff -urN mozilla.orig/suite/branding/Makefile.in mozilla/suite/branding/Makefile.in
--- mozilla.orig/suite/branding/Makefile.in	2009-10-17 15:48:25.000000000 +0200
+++ mozilla/suite/branding/Makefile.in	2009-10-24 13:36:13.000000000 +0200
@@ -91,13 +91,13 @@
 endif
 
 ifeq ($(ICON_DIR),gtk)
-libs:: icons/gtk/seamonkey.png
+libs:: icons/gtk/iceape.png
 	$(INSTALL) $^ $(DIST)/bin/chrome/icons/default
 
-install:: icons/gtk/seamonkey.png
+install:: icons/gtk/iceape.png
 	$(SYSINSTALL) $(IFLAGS1) $^ $(DESTDIR)$(mozappdir)/chrome/icons/default
 
-GARBAGE	+= $(DIST)/bin/chrome/icons/default/seamonkey.png
+GARBAGE	+= $(DIST)/bin/chrome/icons/default/iceape.png
 
 DESKTOP_ICONS += default
 DESKTOP_ICONS_SMALL = $(patsubst %,%16,$(DESKTOP_ICONS))
diff -urN mozilla.orig/suite/browser/browser-prefs.js mozilla/suite/browser/browser-prefs.js
--- mozilla.orig/suite/browser/browser-prefs.js	2009-10-17 15:48:25.000000000 +0200
+++ mozilla/suite/browser/browser-prefs.js	2009-10-24 13:36:13.000000000 +0200
@@ -472,12 +472,12 @@
 
 // Preferences for the Get Add-ons pane
 pref("extensions.getAddons.showPane", true);
-pref("extensions.getAddons.browseAddons", "https://addons.mozilla.org/%LOCALE%/%APP%");
+pref("extensions.getAddons.browseAddons", "https://addons.mozilla.org/%LOCALE%/seamonkey");
 pref("extensions.getAddons.maxResults", 5);
-pref("extensions.getAddons.recommended.browseURL", "https://addons.mozilla.org/%LOCALE%/%APP%/recommended");
-pref("extensions.getAddons.recommended.url", "https://services.addons.mozilla.org/%LOCALE%/%APP%/api/%API_VERSION%/list/featured/all/10/%OS%/%VERSION%");
-pref("extensions.getAddons.search.browseURL", "https://addons.mozilla.org/%LOCALE%/%APP%/search?q=%TERMS%");
-pref("extensions.getAddons.search.url", "https://services.addons.mozilla.org/%LOCALE%/%APP%/api/%API_VERSION%/search/%TERMS%/all/10/%OS%/%VERSION%");
+pref("extensions.getAddons.recommended.browseURL", "https://addons.mozilla.org/%LOCALE%/seamonkey/recommended");
+pref("extensions.getAddons.recommended.url", "https://services.addons.mozilla.org/%LOCALE%/seamonkey/api/%API_VERSION%/list/featured/all/10/%OS%/%VERSION%");
+pref("extensions.getAddons.search.browseURL", "https://addons.mozilla.org/%LOCALE%/seamonkey/search?q=%TERMS%");
+pref("extensions.getAddons.search.url", "https://services.addons.mozilla.org/%LOCALE%/seamonkey/api/%API_VERSION%/search/%TERMS%/all/10/%OS%/%VERSION%");
 
 // Non-symmetric (not shared by extensions) extension-specific [update] preferences
 pref("extensions.getMoreExtensionsURL", "chrome://branding/locale/brand.properties");
diff -urN mozilla.orig/suite/build/nsSuiteModule.cpp mozilla/suite/build/nsSuiteModule.cpp
--- mozilla.orig/suite/build/nsSuiteModule.cpp	2009-10-17 15:48:25.000000000 +0200
+++ mozilla/suite/build/nsSuiteModule.cpp	2009-10-24 13:36:13.000000000 +0200
@@ -98,9 +98,9 @@
     NS_PROFILEMIGRATOR_CONTRACTID,
     nsProfileMigratorConstructor },
   
-  { "SeaMonkey Profile Migrator",
+  { "Iceape Profile Migrator",
     NS_SEAMONKEYPROFILEMIGRATOR_CID,
-    NS_SUITEPROFILEMIGRATOR_CONTRACTID_PREFIX "seamonkey",
+    NS_SUITEPROFILEMIGRATOR_CONTRACTID_PREFIX "iceape",
     nsSeamonkeyProfileMigratorConstructor },
 
   { "Thunderbird Profile Migrator",
diff -urN mozilla.orig/suite/common/src/nsSuiteGlue.js mozilla/suite/common/src/nsSuiteGlue.js
--- mozilla.orig/suite/common/src/nsSuiteGlue.js	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/common/src/nsSuiteGlue.js	2009-10-24 13:36:13.000000000 +0200
@@ -499,7 +499,7 @@
 
 
   // for XPCOM
-  classDescription: "SeaMonkey Suite Glue Service",
+  classDescription: "Iceape Glue Service",
   classID:          Components.ID("{bbbbe845-5a1b-40ee-813c-f84b8faaa07c}"),
   contractID:       "@mozilla.org/suite/suiteglue;1",
 
diff -urN mozilla.orig/suite/common/utilityOverlay.xul mozilla/suite/common/utilityOverlay.xul
--- mozilla.orig/suite/common/utilityOverlay.xul	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/common/utilityOverlay.xul	2009-10-24 13:36:13.000000000 +0200
@@ -358,11 +358,6 @@
                 key="key_openHelp"
                 command="cmd_openHelp"/>
 
-      <menuitem accesskey="&releaseCmd.accesskey;"
-                label="&releaseCmd.label;"
-                id="releaseUrl"
-                oncommand="goReleaseNotes();"/>
-
       <menuseparator id="menu_HelpUpdatesSeparator"/>
 
       <menuitem accesskey="&updateCmd.accesskey;" label="&updateCmd.label;"
diff -urN mozilla.orig/suite/confvars.sh mozilla/suite/confvars.sh
--- mozilla.orig/suite/confvars.sh	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/confvars.sh	2009-10-24 13:36:13.000000000 +0200
@@ -36,8 +36,8 @@
 #
 # ***** END LICENSE BLOCK *****
 
-MOZ_APP_NAME=seamonkey
-MOZ_APP_DISPLAYNAME=SeaMonkey
+MOZ_APP_NAME=iceape
+MOZ_APP_DISPLAYNAME=Iceape
 MOZ_MAIL_NEWS=1
 if [ "$COMM_BUILD" ]; then
   MOZ_LDAP_XPCOM=1
diff -urN mozilla.orig/suite/installer/unix/packages mozilla/suite/installer/unix/packages
--- mozilla.orig/suite/installer/unix/packages	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/installer/unix/packages	2009-10-24 13:36:13.000000000 +0200
@@ -365,7 +365,7 @@
 bin/chrome/icons/default/main-window.png
 bin/chrome/icons/default/main-window16.png
 bin/chrome/icons/default/main-window48.png
-bin/chrome/icons/default/seamonkey.png
+bin/chrome/icons/default/iceape.png
 ; Although venkman-window is an extension, its icon is still suite-specific
 bin/chrome/icons/default/venkman-window.png
 bin/chrome/icons/default/venkman-window16.png
diff -urN mozilla.orig/suite/locales/en-US/chrome/branding/brand.dtd mozilla/suite/locales/en-US/chrome/branding/brand.dtd
--- mozilla.orig/suite/locales/en-US/chrome/branding/brand.dtd	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/locales/en-US/chrome/branding/brand.dtd	2009-10-24 13:36:13.000000000 +0200
@@ -1,5 +1,5 @@
-<!ENTITY  brandFullName         "SeaMonkey">
-<!ENTITY  brandShortName        "SeaMonkey">
-<!ENTITY  vendorShortName       "SeaMonkey">
+<!ENTITY  brandFullName         "Iceape">
+<!ENTITY  brandShortName        "Iceape">
+<!ENTITY  vendorShortName       "Iceape">
 <!ENTITY  sidebarName           "Sidebar">
 <!ENTITY  copyright.string      "Copyright &copy; 2003-2009 Contributors to the Mozilla project.">
diff -urN mozilla.orig/suite/locales/en-US/chrome/branding/brand.properties mozilla/suite/locales/en-US/chrome/branding/brand.properties
--- mozilla.orig/suite/locales/en-US/chrome/branding/brand.properties	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/locales/en-US/chrome/branding/brand.properties	2009-10-24 13:38:12.000000000 +0200
@@ -1,11 +1,11 @@
-brandFullName=SeaMonkey
-brandShortName=SeaMonkey
-vendorShortName=SeaMonkey
+brandFullName=Iceape
+brandShortName=Iceape
+vendorShortName=Iceape
 sidebarName=Sidebar
 extensions.getMoreLocalesURL=http://www.seamonkey-project.org/releases/#l10n
-extensions.getMoreExtensionsURL=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/extensions/
-extensions.getMoreThemesURL=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/themes/
-extensions.getMorePluginsURL=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/plugins/
-spellchecker.dictionaries.download.url=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/dictionaries/
+extensions.getMoreExtensionsURL=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/seamonkey/%VERSION%/extensions/
+extensions.getMoreThemesURL=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/seamonkey/%VERSION%/themes/
+extensions.getMorePluginsURL=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/seamonkey/%VERSION%/plugins/
+spellchecker.dictionaries.download.url=https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/seamonkey/%VERSION%/dictionaries/
 app.releaseNotesURL=http://www.seamonkey-project.org/releases/seamonkey%VERSION%/
-app.vendorURL=http://www.seamonkey-project.org/
+app.vendorURL=http://www.pld-linux.org/
diff -urN mozilla.orig/suite/locales/en-US/chrome/browser/region.properties mozilla/suite/locales/en-US/chrome/browser/region.properties
--- mozilla.orig/suite/locales/en-US/chrome/browser/region.properties	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/locales/en-US/chrome/browser/region.properties	2009-10-24 13:36:43.000000000 +0200
@@ -4,8 +4,8 @@
 #
 # all.js
 #
-browser.startup.homepage=http://www.seamonkey-project.org/start/
-browser.throbber.url=http://www.seamonkey-project.org/
+browser.startup.homepage=about:
+browser.throbber.url=http://www.pld-linux.org/Packages/Iceape
 browser.search.defaulturl=http://www.google.com/search?q=
 
 browser.translation.service=http://translate.google.com/translate?prev=/language_tools&u=
@@ -13,7 +13,7 @@
 
 #config.js
 #
-startup.homepage_override_url=http://www.seamonkey-project.org/releases/seamonkey%VERSION%/
+startup.homepage_override_url=about:
 
 # This is the default set of web based feed handlers shown in the reader
 # selection UI
diff -urN mozilla.orig/suite/locales/en-US/chrome/common/help/cs_nav_prefs_advanced.xhtml mozilla/suite/locales/en-US/chrome/common/help/cs_nav_prefs_advanced.xhtml
--- mozilla.orig/suite/locales/en-US/chrome/common/help/cs_nav_prefs_advanced.xhtml	2009-10-17 15:48:26.000000000 +0200
+++ mozilla/suite/locales/en-US/chrome/common/help/cs_nav_prefs_advanced.xhtml	2009-10-24 13:36:13.000000000 +0200
@@ -386,20 +386,6 @@
         web sites in the current list.</li>
     </ul>
   </li>
-  <li><strong>Automatically check for updates to</strong>:
-    <ul>
-      <li><strong>&brandShortName;</strong>: Select this to be notified when
-        new versions of &brandShortName; are available. Your personal
-        information is not shared when verifying your version of
-        &brandShortName;. Choose whether you want &brandShortName; to do a
-        <strong>daily</strong> or a <strong>weekly</strong> check for new
-        versions.</li>
-      <li><strong>Installed add-ons</strong>: Select this to be notified when a
-        new version of one of your installed add-ons is available. Choose
-        whether you want &brandShortName; to do a <strong>daily</strong> or a
-        <strong>weekly</strong> check for new versions.</li>
-    </ul>
-  </li>
   <li><strong>Add-on Manager</strong>: Click this to open the
     Add-on Manager, where you can view and manage all your installed
     extensions and themes.</li>
diff -urN mozilla.orig/suite/locales/en-US/chrome/common/search/default.htm mozilla/suite/locales/en-US/chrome/common/search/default.htm
--- mozilla.orig/suite/locales/en-US/chrome/common/search/default.htm	2009-10-17 15:48:27.000000000 +0200
+++ mozilla/suite/locales/en-US/chrome/common/search/default.htm	2009-10-24 13:36:13.000000000 +0200
@@ -6,7 +6,7 @@
 </HEAD>
 
 <BODY>
-	<CENTER>SeaMonkey Search</CENTER>
+	<CENTER>Iceape Search</CENTER>
 </BODY>
 
 </HTML>
diff -urN mozilla.orig/suite/locales/en-US/chrome/mailnews/region.properties mozilla/suite/locales/en-US/chrome/mailnews/region.properties
--- mozilla.orig/suite/locales/en-US/chrome/mailnews/region.properties	2009-10-17 15:48:27.000000000 +0200
+++ mozilla/suite/locales/en-US/chrome/mailnews/region.properties	2009-10-24 13:36:13.000000000 +0200
@@ -2,9 +2,9 @@
 # messenger.properties
 # mailnews.js
 mailnews.start_page.url=chrome://messenger/content/start.xhtml
-messenger.throbber.url=http://www.seamonkey-project.org/
-compose.throbber.url=http://www.seamonkey-project.org/
-addressbook.throbber.url=http://www.seamonkey-project.org/
+messenger.throbber.url=http://www.pld-linux.org/Packages/Iceape
+compose.throbber.url=http://www.pld-linux.org/Packages/Iceape
+addressbook.throbber.url=http://www.pld-linux.org/Packages/Iceape
 # To make mapit buttons to disappear in the addressbook, specify empty string.  For example:
 # mail.addr_book.mapit_url.format=
 # The format for "mail.addr_book.mapit_url.format" is:
diff -urN mozilla.orig/suite/locales/en-US/chrome/mailnews/start.dtd mozilla/suite/locales/en-US/chrome/mailnews/start.dtd
--- mozilla.orig/suite/locales/en-US/chrome/mailnews/start.dtd	2009-10-17 15:48:27.000000000 +0200
+++ mozilla/suite/locales/en-US/chrome/mailnews/start.dtd	2009-10-24 13:36:13.000000000 +0200
@@ -38,7 +38,7 @@
 <!ENTITY startpage.title "Welcome to &brandShortName; Mail &amp; Newsgroups">
 <!--# LOCALIZATION NOTE (headline.label): the URL for <a id="vendorURL"> is fetched from brand.properties -->
 <!ENTITY headline.label
-'Welcome to <a id="vendorURL" href="">&brandShortName;</a> Mail &amp; Newsgroups!'>
+'Welcome to &brandShortName; Mail &amp; Newsgroups!'>
 <!ENTITY description.label
 "&brandShortName; Mail &amp; Newsgroups is a powerful open-source mail and news
 client, supporting advanced junk mail detection and other useful features.">
diff -urN mozilla.orig/suite/locales/generic/profile/bookmarks.html.in mozilla/suite/locales/generic/profile/bookmarks.html.in
--- mozilla.orig/suite/locales/generic/profile/bookmarks.html.in	2009-10-17 15:48:27.000000000 +0200
+++ mozilla/suite/locales/generic/profile/bookmarks.html.in	2009-10-24 13:36:13.000000000 +0200
@@ -43,6 +43,14 @@
             <DT><A HREF="http://forums.mozillazine.org/viewforum.php?f=40">@seamonkey_support@</A>
         </DL><p>
     </DL><p>
+    <DT><H3>PLD Linux Distribution</H3>
+    <DL><p>
+        <DT><A HREF="http://www.pld-linux.org/">PLD Home Page</A>
+        <DT><A HREF="http://www.pld-users.org/">PLD Users Wiki</A>
+        <DT><A HREF="http://docs.pld-linux.org/">PLD Documentation in english</A>
+        <DT><A HREF="http://pl.docs.pld-linux.org">PLD Documentation in polish</A>
+        <DT><A HREF="http://planet.pld-linux.org/">Planet PLD</A>
+    </DL><p>
 #include bookmarks.extra
     <HR>
 </DL><p>
diff -urN mozilla.orig/suite/mailnews/start.xhtml mozilla/suite/mailnews/start.xhtml
--- mozilla.orig/suite/mailnews/start.xhtml	2009-10-17 15:48:27.000000000 +0200
+++ mozilla/suite/mailnews/start.xhtml	2009-10-24 13:36:13.000000000 +0200
@@ -73,8 +73,6 @@
 <h2>&dict.title;</h2>
 <p>&dict_intro.label;</p>
 <p>&dict_info.label;</p>
-<h2>&info.title;</h2>
-<p>&info_bugs.label;</p>
 </div>
 
 <script type="application/javascript">
diff -urN mozilla.orig/suite/themes/classic/install.rdf mozilla/suite/themes/classic/install.rdf
--- mozilla.orig/suite/themes/classic/install.rdf	2009-10-17 15:48:27.000000000 +0200
+++ mozilla/suite/themes/classic/install.rdf	2009-10-24 13:36:13.000000000 +0200
@@ -21,7 +21,7 @@
     </em:targetApplication>
 
     <!-- Front End MetaData -->
-    <em:name>SeaMonkey Default Theme</em:name>
+    <em:name>Iceape Default Theme</em:name>
     <em:description>This theme uses styles and colors from the system to fit in with other applications.</em:description>
 
     <!-- EXTENSION AUTHORS!
diff -urN mozilla.orig/suite/themes/modern/install.rdf mozilla/suite/themes/modern/install.rdf
--- mozilla.orig/suite/themes/modern/install.rdf	2009-10-17 15:48:28.000000000 +0200
+++ mozilla/suite/themes/modern/install.rdf	2009-10-24 13:36:13.000000000 +0200
@@ -20,7 +20,7 @@
     </em:targetApplication>
 
     <!-- Front End MetaData -->
-    <em:name>SeaMonkey Modern</em:name>
+    <em:name>Iceape Modern</em:name>
     <em:description>A contemporary theme for all components.</em:description>
 
     <!-- EXTENSION AUTHORS!
