--- seamonkey-1.1.11/mozilla/xpcom/glue/standalone/nsGlueLinkingDlopen.cpp	2006-02-22 22:57:09.000000000 +0100
+++ seamonkey-1.1.11/mozilla/xpcom/glue/standalone/nsGlueLinkingDlopen.cpp.fixed	2008-09-23 21:15:21.000000000 +0200
@@ -86,8 +86,8 @@
 GetFrozenFunctionsFunc
 XPCOMGlueLoad(const char *xpcomFile)
 {
-    char xpcomDir[MAXPATHLEN];
-    if (realpath(xpcomFile, xpcomDir)) {
+    char * xpcomDir;
+    if (xpcomDir = realpath(xpcomFile, NULL)) {
         char *lastSlash = strrchr(xpcomDir, '/');
         if (lastSlash) {
             *lastSlash = '\0';
@@ -98,6 +98,7 @@
 
             sXULLibHandle = dlopen(xpcomDir, RTLD_GLOBAL | RTLD_LAZY);
         }
+	free( xpcomDir );
     }
 
     // RTLD_DEFAULT is not defined in non-GNU toolchains, and it is
