diff --git a/suite/browser/browser-prefs.js b/suite/browser/browser-prefs.js
index 1e3799e..ae4cb1f 100644
--- a/suite/browser/browser-prefs.js
+++ b/suite/browser/browser-prefs.js
@@ -41,7 +41,7 @@
  * modules/libpref/src/init/all.js
  */
 
-pref("startup.homepage_override_url","chrome://navigator-region/locale/region.properties");
+pref("startup.homepage_override_url","");
 pref("general.skins.selectedSkin", "classic/1.0");
 
 pref("browser.chromeURL","chrome://navigator/content/navigator.xul");
@@ -110,7 +110,7 @@ pref("browser.findbar.enabled", true);
 pref("browser.doorhanger.enabled", true);
 
 // search engines URL
-pref("browser.search.searchEnginesURL", "https://addons.mozilla.org/%LOCALE%/%APP%/search-engines/");
+pref("browser.search.searchEnginesURL", "https://addons.mozilla.org/%LOCALE%/seamonkey/search-engines/");
 
 // pointer to the default engine name
 pref("browser.search.defaultenginename", "chrome://communicator-region/locale/region.properties");
@@ -563,7 +563,7 @@ pref("extensions.blocklist.interval", 86400);
 pref("extensions.blocklist.level", 2);
 pref("extensions.blocklist.url", "https://addons.mozilla.org/blocklist/3/%APP_ID%/%APP_VERSION%/%PRODUCT%/%BUILD_ID%/%BUILD_TARGET%/%LOCALE%/%CHANNEL%/%OS_VERSION%/%DISTRIBUTION%/%DISTRIBUTION_VERSION%/%PING_COUNT%/%TOTAL_PING_COUNT%/%DAYS_SINCE_LAST_PING%/");
 pref("extensions.blocklist.detailsURL", "https://www.mozilla.com/%LOCALE%/blocklist/");
-pref("extensions.blocklist.itemURL", "https://addons.mozilla.org/%LOCALE%/%APP%/blocked/%blockID%");
+pref("extensions.blocklist.itemURL", "https://addons.mozilla.org/%LOCALE%/seamonkey/blocked/%blockID%");
 
 // Update preferences for installed Extensions and Themes.
 // Symmetric (can be overridden by individual extensions),
@@ -587,11 +587,11 @@ pref("extensions.autoDisableScopes", 15);
 // Preferences for AMO integration
 pref("extensions.getAddons.cache.enabled", true);  // also toggles personalized recommendations
 pref("extensions.getAddons.maxResults", 15);
-pref("extensions.getAddons.get.url", "https://services.addons.mozilla.org/%LOCALE%/%APP%/api/%API_VERSION%/search/guid:%IDS%?src=seamonkey&appOS=%OS%&appVersion=%VERSION%");
-pref("extensions.getAddons.getWithPerformance.url", "https://services.addons.mozilla.org/%LOCALE%/%APP%/api/%API_VERSION%/search/guid:%IDS%?src=seamonkey&appOS=%OS%&appVersion=%VERSION%&tMain=%TIME_MAIN%&tFirstPaint=%TIME_FIRST_PAINT%&tSessionRestored=%TIME_SESSION_RESTORED%");
-pref("extensions.getAddons.search.browseURL", "https://addons.mozilla.org/%LOCALE%/%APP%/search?q=%TERMS%");
-pref("extensions.getAddons.search.url", "https://services.addons.mozilla.org/%LOCALE%/%APP%/api/%API_VERSION%/search/%TERMS%/all/%MAX_RESULTS%/%OS%/%VERSION%/%COMPATIBILITY_MODE%?src=seamonkey");
-pref("extensions.webservice.discoverURL", "https://services.addons.mozilla.org/%LOCALE%/%APP%/discovery/pane/%VERSION%/%OS%");
+pref("extensions.getAddons.get.url", "https://services.addons.mozilla.org/%LOCALE%/seamonkey/api/%API_VERSION%/search/guid:%IDS%?src=seamonkey&appOS=%OS%&appVersion=%VERSION%");
+pref("extensions.getAddons.getWithPerformance.url", "https://services.addons.mozilla.org/%LOCALE%/seamonkey/api/%API_VERSION%/search/guid:%IDS%?src=seamonkey&appOS=%OS%&appVersion=%VERSION%&tMain=%TIME_MAIN%&tFirstPaint=%TIME_FIRST_PAINT%&tSessionRestored=%TIME_SESSION_RESTORED%");
+pref("extensions.getAddons.search.browseURL", "https://addons.mozilla.org/%LOCALE%/seamonkey/search?q=%TERMS%");
+pref("extensions.getAddons.search.url", "https://services.addons.mozilla.org/%LOCALE%/seamonkey/api/%API_VERSION%/search/%TERMS%/all/%MAX_RESULTS%/%OS%/%VERSION%/%COMPATIBILITY_MODE%?src=seamonkey");
+pref("extensions.webservice.discoverURL", "https://services.addons.mozilla.org/%LOCALE%/seamonkey/discovery/pane/%VERSION%/%OS%");
 
 // getMoreThemes is used by our UI under our switch theme menu
 pref("extensions.getMoreThemesURL", "chrome://branding/locale/brand.properties");
diff --git a/suite/locales/en-US/chrome/browser/region.properties b/suite/locales/en-US/chrome/browser/region.properties
index f56a77e..56e49f2 100644
--- a/suite/locales/en-US/chrome/browser/region.properties
+++ b/suite/locales/en-US/chrome/browser/region.properties
@@ -2,8 +2,8 @@
 #
 # all.js
 #
-browser.startup.homepage=http://www.seamonkey-project.org/start/
-browser.throbber.url=http://www.seamonkey-project.org/
+browser.startup.homepage=about:
+browser.throbber.url=http://www.pld-linux.org/Packages/Iceape
 browser.search.defaulturl=http://www.google.com/search?q=
 
 browser.translation.service=http://translate.google.com/translate?prev=/language_tools&u=
@@ -11,7 +11,7 @@ browser.translation.serviceDomain=translate.google.com
 
 #config.js
 #
-startup.homepage_override_url=http://www.seamonkey-project.org/releases/seamonkey%VERSION%/
+startup.homepage_override_url=about:
 
 # This is the default set of web based feed handlers shown in the reader
 # selection UI
diff --git a/suite/locales/en-US/chrome/branding/brand.dtd b/suite/locales/en-US/chrome/branding/brand.dtd
index 6459fba..946fab4 100644
--- a/suite/locales/en-US/chrome/branding/brand.dtd
+++ b/suite/locales/en-US/chrome/branding/brand.dtd
@@ -1,4 +1,4 @@
-<!ENTITY  brandFullName         "SeaMonkey">
-<!ENTITY  brandShortName        "SeaMonkey">
-<!ENTITY  vendorShortName       "SeaMonkey">
+<!ENTITY  brandFullName         "Iceape">
+<!ENTITY  brandShortName        "Iceape">
+<!ENTITY  vendorShortName       "PLD">
 <!ENTITY  sidebarName           "Sidebar">
diff --git a/suite/locales/en-US/chrome/branding/brand.properties b/suite/locales/en-US/chrome/branding/brand.properties
index f068049..f4a29a9 100644
--- a/suite/locales/en-US/chrome/branding/brand.properties
+++ b/suite/locales/en-US/chrome/branding/brand.properties
@@ -1,6 +1,6 @@
-brandFullName=SeaMonkey
-brandShortName=SeaMonkey
-vendorShortName=SeaMonkey
+brandFullName=Iceape
+brandShortName=Iceape
+vendorShortName=PLD
 sidebarName=Sidebar
 extensions.getMoreThemesURL=https://addons.mozilla.org/%LOCALE%/%APP%/themes
 extensions.getPersonasURL=https://addons.mozilla.org/%LOCALE%/%APP%/personas
diff --git a/suite/themes/classic/install.rdf b/suite/themes/classic/install.rdf
index 8b77d5d..2ea4fb9 100644
--- a/suite/themes/classic/install.rdf
+++ b/suite/themes/classic/install.rdf
@@ -21,7 +21,7 @@
     </em:targetApplication>
 
     <!-- Front End MetaData -->
-    <em:name>SeaMonkey Default Theme</em:name>
+    <em:name>Iceape Default Theme</em:name>
     <em:description>This theme uses styles and colors from the system to fit in with other applications.</em:description>
 
     <!-- EXTENSION AUTHORS!
diff --git a/suite/themes/modern/install.rdf b/suite/themes/modern/install.rdf
index d20cb5b..4fa63af 100644
--- a/suite/themes/modern/install.rdf
+++ b/suite/themes/modern/install.rdf
@@ -20,7 +20,7 @@
     </em:targetApplication>
 
     <!-- Front End MetaData -->
-    <em:name>SeaMonkey Modern</em:name>
+    <em:name>Iceape Modern</em:name>
     <em:description>A contemporary theme for all components.</em:description>
 
     <!-- EXTENSION AUTHORS!
--- comm-release/suite/installer/package-manifest.in~	2012-06-05 07:50:28.000000000 +0200
+++ comm-release/suite/installer/package-manifest.in	2012-06-06 14:55:27.764056749 +0200
@@ -480,7 +480,6 @@
 @BINPATH@/chrome/icons/default/places.png
 @BINPATH@/chrome/icons/default/places16.png
 @BINPATH@/chrome/icons/default/places48.png
-@BINPATH@/chrome/icons/default/seamonkey.png
 #ifdef MOZ_UPDATER
 @BINPATH@/icons/updater.png
 #endif
