--- iceape-2.0/configure.in.orig	2009-09-03 19:44:56.000000000 +0200
+++ iceape-2.0/configure.in	2009-09-19 12:59:36.000000000 +0200
@@ -7910,7 +7910,7 @@
         mkdir "directory/c-sdk/ldap"
     fi
 
-    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --without-nss --with-mozilla"
+    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-nss --with-mozilla"
     if test -z "$MOZ_DEBUG"; then
         ac_configure_args="$ac_configure_args --disable-debug"
     fi
--- iceape-2.0/configure.orig	2009-09-03 19:50:42.000000000 +0200
+++ iceape-2.0/configure	2009-09-19 12:59:51.000000000 +0200
@@ -21460,7 +21460,7 @@
         mkdir "directory/c-sdk/ldap"
     fi
 
-    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --without-nss --with-mozilla"
+    ac_configure_args="$_SUBDIR_CONFIG_ARGS --prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-dist-prefix=$MOZ_BUILD_ROOT/mozilla/dist --with-nss --with-mozilla"
     if test -z "$MOZ_DEBUG"; then
         ac_configure_args="$ac_configure_args --disable-debug"
     fi
