--- ./mozilla/modules/libpr0n/encoders/png/nsPNGEncoder.cpp.org	2010-01-04 22:04:00.000000000 +0100
+++ ./mozilla/modules/libpr0n/encoders/png/nsPNGEncoder.cpp	2010-02-16 14:21:57.621636697 +0100
@@ -135,7 +135,7 @@ NS_IMETHODIMP nsPNGEncoder::StartImageEn
 
   // initialize
   mPNG = png_create_write_struct(PNG_LIBPNG_VER_STRING,
-                                 png_voidp_NULL,
+                                 NULL,
                                  ErrorCallback,
                                  ErrorCallback);
   if (! mPNG)
