--- mozilla/xpfe/bootstrap/browser-prefs.js	2006-12-22 14:51:33.000000000 +0100
+++ mozilla.orig/xpfe/bootstrap/browser-prefs.js	2008-06-24 21:50:31.000000000 +0200
@@ -221,7 +221,7 @@
 pref("alerts.height", 50);
 
 // update notifications prefs
-pref("update_notifications.enabled", true);
+pref("update_notifications.enabled", false);
 pref("update_notifications.provider.0.frequency", 7); // number of days
 pref("update_notifications.provider.0.datasource", "chrome://communicator-region/locale/region.properties");
 
