--- iceape-2.20/comm-release/suite/installer/removed-files.in.orig	2013-08-15 17:04:13.800677008 +0200
+++ iceape-2.20/comm-release/suite/installer/removed-files.in	2013-08-15 17:08:14.499464514 +0200
@@ -192,9 +192,6 @@
 D3DCompiler_42.dll
 d3dx9_42.dll
 d3dx9_43.dll
-#ifdef UNIX_BUT_NOT_MAC
-components/nsFilePicker.js
-#endif
 #ifndef MOZ_CRASHREPORTER
 #ifdef XP_MACOSX
 crashreporter.app/
@@ -617,34 +614,9 @@
   chrome/gloda.jar
   chrome/nonlocalized.manifest
   chrome/localized.manifest
-  components/addonManager.js
   components/Aitc.js
-  components/AlarmsManager.js
-  components/amContentHandler.js
-  components/amWebInstallListener.js
-  components/BrowserElementParent.js
-  components/ColorAnalyzer.js
-  components/ConsoleAPI.js
-  components/ContactManager.js
-  components/contentAreaDropListener.js
-  components/contentSecurityPolicy.js
-  components/crypto-SDR.js
-  components/FeedConverter.js
-  components/FeedProcessor.js
-  components/FeedWriter.js
-  components/glautocomp.js
   components/GPSDGeolocationProvider.js
-  components/jsconsole-clhandler.js
-  components/jsmimeemitter.js
-  components/mdn-service.js
-  components/msgAsyncPrompter.js
-  components/NetworkGeolocationProvider.js
   components/NotificationsComponents.manifest
-  components/newsblog.js
-  components/nsAbAutoCompleteMyDomain.js
-  components/nsAbAutoCompleteSearch.js
-  components/nsAbLDAPAttributeMap.js
-  components/nsAbout.js
   components/nsAboutCertError.js
   components/nsAboutData.js
   components/nsAboutFeeds.js
@@ -653,69 +625,10 @@
   components/nsAboutSessionRestore.js
   components/nsAboutSyncTabs.js
   components/nsBadCertHandler.js
-  components/nsBlocklistService.js
-  components/nsBrowserContentHandler.js
-  components/nsComposerCmdLineHandler.js
-  components/nsContentDispatchChooser.js
-  components/nsContentPrefService.js
-  components/nsDefaultCLH.js
-  components/nsDOMIdentity.js
-  #ifdef UNIX_BUT_NOT_MAC
-    components/nsFilePicker.js
-  #endif
-  components/nsFormAutoComplete.js
-  components/nsFormHistory.js
-  components/nsHandlerService.js
-  components/nsHelperAppDlg.js
-  components/nsIDService.js
-  components/nsINIProcessor.js
-  components/nsInputListAutoComplete.js
-  components/nsLDAPProtocolHandler.js
-  components/nsLivemarkService.js
-  components/nsLoginInfo.js
-  components/nsLoginManager.js
-  components/nsLoginManagerPrompter.js
-  components/nsMailNewsCommandLineHandler.js
-  components/nsMsgTraitService.js
-  components/nsPlacesAutoComplete.js
   components/nsPlacesDBFlush.js
-  components/nsPlacesExpiration.js
-  components/nsPrompter.js
-  components/nsSearchService.js
-  components/nsSearchSuggestions.js
-  components/nsSessionStartup.js
-  components/nsSessionStore.js
-  components/nsSetDefault.js
-  components/nsSidebar.js
-  components/nsSMTPProtocolHandler.js
-  components/nsSuiteDownloadManagerUI.js
-  components/nsSuiteGlue.js
-  components/nsTaggingService.js
   components/nsTryToClose.js
-  components/nsTypeAheadFind.js
   components/nsUpdateService.js
   components/nsUpdateServiceStub.js
-  components/nsUpdateTimerManager.js
-  #ifdef MOZ_URL_CLASSIFIER
-    components/nsUrlClassifierHashCompleter.js
-    components/nsUrlClassifierLib.js
-    components/nsUrlClassifierListManager.js
-  #endif
-  components/nsURLFormatter.js
-  components/nsWebHandlerApp.js
-  components/offlineStartup.js
-  components/PlacesCategoriesStarter.js
-  components/SettingsManager.js
-  components/SiteSpecificUserAgent.js
-  components/smileApplication.js
-  components/smime-service.js
-  components/storage-Legacy.js
-  components/storage-mozStorage.js
-  components/TCPSocket.js
-  components/txEXSLTRegExFunctions.js
-  components/Weave.js
-  components/Webapps.js
-  components/WebContentConverter.js
   components/browser.xpt
   components/binary.manifest
   components/interfaces.manifest
